tomee/tomee-microprofile/mp-common/src/main/java/org/apache/tomee/microprofile/TomEEMicroProfileListener.java

```diff
@@ -66,6 +71,8 @@ public class TomEEMicroProfileListener {
         "io.smallrye.faulttolerance.FaultToleranceExtension",
         };
 
+    private final Map<AppInfo, Index> indexCache = new HashMap<>();
```

Comment:
should still be concurrent ;)

rest looks ok

Reply:
done 👍